.github/workflows/deploy.yml

```
@@ -11,23 +11,29 @@ jobs:
     env:
-      # VITE_APP_API_URL, VITE_APP_WS_URL, VITE_ONE_SIGNAL_APP_ID는 이미 CI 빌드 과정에서 아티팩트 내에 포함
+      VITE_APP_API_URL: ${{ secrets.VITE_APP_API_URL }}
+      VITE_APP_WS_URL: ${{ secrets.VITE_APP_WS_URL }}
       AWS_S3_BUCKET: ${{ secrets.AWS_S3_BUCKET }}
       CLOUDFRONT_DISTRIBUTION_ID: ${{ secrets.CLOUDFRONT_DISTRIBUTION_ID }}
       AWS_REGION: ${{ secrets.AWS_REGION }}
+      VITE_ONE_SIGNAL_APP_ID: ${{ secrets.VITE_ONE_SIGNAL_APP_ID }}
 
     steps:
       - name: Checkout repository
         uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)